### PR TITLE
Shepherd Hunger Fix

### PR DESCRIPTION
**Pull request by Asherslab**
# Changes proposed in this pull request:
- Reduces hunger reduction by half 

Review please

### CODE REVIEW

File: src/main/java/com/minecolonies/coremod/entity/ai/citizen/herders/AbstractEntityAIHerder.java [hunk 1]
```diff
@@ -370,7 +371,7 @@ public int getMaxAnimalMultiplier()
      */
     public abstract Class<T> getAnimalClass();
 
-    /**
+    /**d
```
Comment (ToMe25) [inline src/main/java/com/minecolonies/coremod/entity/ai/citizen/herders/AbstractEntityAIHerder.java]: is this 'd' meant to be here?

File: src/main/java/com/minecolonies/coremod/entity/ai/citizen/herders/AbstractEntityAIHerder.java [hunk 2]
```diff
@@ -319,7 +319,7 @@ private IAIState pickupItems()
             }
         }
 
-        incrementActionsDoneAndDecSaturation();
+        incrementActionsDone();
```
Comment (Raycoms) [inline src/main/java/com/minecolonies/coremod/entity/ai/citizen/herders/AbstractEntityAIHerder.java]: at least make this a "continous action" decrement of saturation (eggs etc that are picked up are a action too)

Comment (Asherslab) [reply src/main/java/com/minecolonies/coremod/entity/ai/citizen/herders/AbstractEntityAIHerder.java]: what are you talking about?

**Review by ToMe25 — APPROVED** [review]
Seems fine to me.

**Comment by Raycoms** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/ldtteam/minecolonies/pull-request/4900/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ldtteam/minecolonies)

**Pull request merged**